PowerPointLabs/PowerPointLabs/PasteLab/PasteLabMain.cs

```diff
@@ -237,5 +260,18 @@ public static void PasteToPosition(Models.PowerPointSlide slide, bool clipboardI
                 }
             }
         }
+
+        private static ShapeRange RemovePlaceholders(PowerPointSlide slide, ShapeRange shapes)
```

Comment:
quick question, it feels strange the methods are put into Graphics.cs,
should it be put into shapes' related util, what do you think